curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.md

```diff
@@ -21,12 +21,13 @@ const arr = [
   [[10, 11, 12], 13, 14]
 ];
 
-arr[3];
-arr[3][0];
-arr[3][0][1];
+const subarray_3 = arr[3];
+const subarray_3_0 = arr[3][0];
+const entry_3_0_1 = arr[3][0][1];
```

Comment:
I think we should follow standard naming conventions here.

These should either be `subarray3`, `subarray30`, `entry301`, or `subarrayThree`, `subarrayThreeZero`, `entryThreeZeroOne`. I think I prefer the second.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
I think the first is unclear as to the indexing, which is why I added underscores.

I thought JS had the convention "don't use underscores unless it clearly improves readability".

I think using words for the numbers makes this harder to read.

maybe
```javascript
const subarray = arr[3];
const nestedSubarray = arr[3][0];
const element = arr[3][0][1];
```

Reply:
That works even better, I think.